sdk/security/keyvault/azadmin/backup/utils_test.go

```diff
@@ -7,25 +7,21 @@
 package backup_test
 
 import (
-	"context"
 	"encoding/json"
-	"fmt"
 	"os"
 	"regexp"
 	"testing"
-	"time"
 
 	"github.com/Azure/azure-sdk-for-go/sdk/azcore"
-	"github.com/Azure/azure-sdk-for-go/sdk/azcore/policy"
-	"github.com/Azure/azure-sdk-for-go/sdk/azidentity"
 	"github.com/Azure/azure-sdk-for-go/sdk/internal/recording"
+	azcred "github.com/Azure/azure-sdk-for-go/sdk/internal/test/credential"
 	"github.com/Azure/azure-sdk-for-go/sdk/security/keyvault/azadmin/backup"
 	"github.com/stretchr/testify/require"
 )
 
 const recordingDirectory = "sdk/security/keyvault/azadmin/testdata"
-const fakeHsmURL = "https://fakehsm.managedhsm.azure.net/"
-const fakeBlobURL = "https://fakestorageaccount.blob.core.windows.net/backup"
+const fakeHsmURL = "https://Sanitized.managedhsm.azure.net/"
+const fakeBlobURL = "https://Sanitized.blob.core.windows.net/backup"
```

Comment:
I'm thinking if you drop the path segment, you can rely on the proxy's built-in URL sanitizer
```suggestion
const fakeBlobURL = fmt.Sprintf("https://%s.blob.core.windows.net", recording.SanitizedValue)
```

Reply:
tried it out, and need to leave backup in for the recordings.

Reply:
I'm too curious not to ask: Why? I naively expected the recorded path not to matter because our client doesn't do anything with it, so my mental model needs revision 😊

Reply:
the blob URL is in the format: https://<storage-account>.blob.core.windows.net/<container>, if we don't include the container name in the fake URL, we'll get malformed recordings